Sources/Toast/ToastStyle.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+extension Warp {
+    public enum ToastStyle: Equatable, CaseIterable {
+        /// Indicate a severe problem or error.
```

Comment:
Please use these for documentations, you can find them [here](https://www.figma.com/file/nkiRpuVu6XRfvY96BA80H8/Components-overview?type=design&node-id=377-23910&mode=design&t=Xt8LzafYSVzUNSGi-0)
This way we will sync our documentation across platforms, unless you have copied them from somewhere that makes more sense.
<img width="594" alt="Screenshot 2024-01-31 at 13 03 51" src="https://github.com/warp-ds/warp-ios/assets/89784012/6caeb6a5-1b8f-4e3f-955e-b755217a0ee5">

Reply:
Ah nice! Then I don't have to come up with descriptions myself when someone who actually knows what they are doing has already done the work 😅